### PR TITLE
SOL-117019: EOL OpenSSL 1.1.1

### PR DESCRIPTION
**Pull request by cjwmorgan-sol**

### CODE REVIEW

File: update_ccsmp.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Needs go on the path, probably:
+# export PATH=$PATH:/usr/local/go/bin
+# Blindly following the instructions at https://sol-jira.atlassian.net/wiki/spaces/CPD/pages/2868052115/Golang+Process+and+Tools#Updating-CCSMP
+# Worked once, so saving for posterity.
+
+LOAD=/home/public/RND/loads/ccsmp/trunk/100.0.0.924/
```
Comment (cjwmorgan-sol) [inline update_ccsmp.sh:8]: This is a public repository this is not allows in the git commit or history please remove

Comment (cjwmorgan-sol) [reply update_ccsmp.sh:8]: Honestly I do no think we should put this script in this project please remove.

**Review by cjwmorgan-sol — COMMENTED** [review]
@gszol Please address the script security issue.

File: update_ccsmp.sh (hunk 1 above)
Comment (oodigie) [reply update_ccsmp.sh:8]: I agree, we shouldn't be exposing the internal directory structure of our dev servers. We can put the script in the Confluence page that described the update process.

Comment (gszol) [reply update_ccsmp.sh]: I deleted the script.

**Review by cjwmorgan-sol — COMMENTED** [review]
Looks good to me.

**Review by oodigie — APPROVED** [review]
LGTM!

**Pull request merged**